critiquebrainz/db/review.py

```diff
@@ -417,8 +433,15 @@ def get_reviews_list(connection, *, inc_drafts=False, inc_hidden=False, entity_i
         filter_data["entity_id"] = entity_id
 
     if entity_type is not None:
-        filters.append("entity_type = :entity_type")
-        filter_data["entity_type"] = entity_type
+        if entity_type == 'musicbrainz':
```

Comment:
This is a great additional feature, but it should be documented everwhere it can be used - both in the docstring for this db method, and also in the API documentation if this can be used directly from the API

Reply:
I agree. I'll add it in the documentation